config.toml

```diff
@@ -50,7 +50,40 @@ dockerImage = 'mojaloop/central-ledger:latest'
 # Ignored when `mode != "docker"`
 # Override by setting `dockerImages` env variable
 ##
-dockerImages = []
+dockerImages = [
+  "mojaloop/account-lookup-service:v8.7.0",
+  "mojaloop/als-oracle-pathfinder:v8.7.1",
+  "mojaloop/bulk-api-adapter:v8.7.0-snapshot",
+  "mojaloop/central-end-user-registry:v2.4",
+  "mojaloop/central-event-processor:v8.6.0",
+  "mojaloop/central-event-processor:v8.7.0",
+  "mojaloop/central-ledger:v8.7.1",
+  "mojaloop/central-settlement:v8.6.0",
+  "mojaloop/email-notifier:v8.7.0",
+  "mojaloop/event-sidecar:v8.7.0",
+  "mojaloop/event-sidecar:v8.7.1",
+  "mojaloop/event-stream-processor:v8.7.2-snapshot",
+  "mojaloop/finance-portal-backend-service:v0.0.1",
+  "mojaloop/finance-portal-backend-service:v0.0.5",
+  "mojaloop/finance-portal-ui:v0.0.3",
+  "mojaloop/ml-api-adapter:v8.7.2",
+  "mojaloop/operator-settlement:latest",
+  "mojaloop/quoting-service:v8.7.0-snapshot",
+  "mojaloop/settlement-management:latest",
+  "mojaloop/simulator:v8.7.1",
+  "mojaloop/transaction-requests-service:v8.7.0"
+]
```

Comment:
should we start adding mojaloop-simulator, sdk-scheme-adapter, sdk-standard-components to the list as well?

Reply:
This doesn't really get run in this form, since in helm we use the `format-docker-images.sh` script to get the list of docker images to scan, which then overrides this list.

I just added it in so I could test the whitelist functionality, and left it as an example